apps/ecparam.c

```diff
@@ -327,10 +327,6 @@ int ecparam_main(int argc, char **argv)
 
         if ((point = EC_GROUP_get0_generator(group)) == NULL)
             goto end;
-        if (!EC_POINT_point2bn(group, point,
-                               EC_GROUP_get_point_conversion_form(group),
-                               ec_gen, NULL))
-            goto end;
```

Comment:
You're removing this conversion call, but the generator in ec_gen is printed down below.

Reply:
I am not too concerned about this since.
There are other active PR's that address removing this entire code block completely.

Reply:
The rebase got rid of your question since the PR that removed these code blocks has been merged..